.gitignore

```diff
@@ -1,3 +1,4 @@
 *.log
 node_modules
 coverage*
+.idea
```

Comment:
let's no include IDE specific files in favor of users just using their `.gitignore_global`
```suggestion

```

Reply:
Ah yes, I agree. I just learned about `.gitignore_global` a couple weeks ago, then promptly forgot about it 😆